reddit_liveupdate/public/static/css/liveupdate-public.less

```diff
@@ -0,0 +1,123 @@
+// CSS in here is imported into reddit's main CSS file, for inclusion of live features into reddit proper
```

Comment:
Huh, I don't know why I didn't catch that. Thanks Neil.

I've been digging into this and I don't see a great way to maintain plugin separation and compile into reddit.less. Something I'm considering now is a generated `plugins.less` file that we always import that we generate on `make` which will contain `@import`s for all the plugins that would require injecting into reddit.less. That would work, but as I dig into it it's looking a bit gnarly in the main `Makefile`.

Can you think of any other approaches? It doesn't look like less supports importing directories or wildcards (because then the JS version would have trouble).